src/components/CurrentRefinements.vue

```diff
@@ -5,28 +5,41 @@
   >
     <slot
       :refine="state.refine"
-      :items="refinements"
+      :items="state.items"
       :createURL="state.createURL"
     >
       <ul :class="suit('list')">
         <li
-          v-for="item in refinements"
-          :key="item.key"
+          v-for="item in state.items"
+          :key="item.attribute"
+          :class="suit('item')"
         >
-          <slot
-            name="item"
-            :refine="state.refine"
-            :item="item"
-            :createURL="() => state.createURL(item.value)"
+          <span :class="suit('label')">{{ item.attribute | capitalize }}: </span>
+          <span
+            v-for="refinement in item.refinements"
+            :key="createItemKey(refinement)"
+            :class="suit('category')"
           >
-            <span :class="suit('item')">
-              <span :class="suit('label')">{{ item.attribute | capitalize }}: {{ item.label }}</span>
+            <slot
+              name="item"
```

Comment:
The slots could be moved on the `li`, users might want to be able to customize the label.

Reply:
Could probably be two slots, one for the inner item, one for the containing attribute